tools/spectral/ipa/rulesets/functions/utils/resourceEvaluation.js

```diff
@@ -0,0 +1,63 @@
+export function isChild(path) {
+  return path.endsWith('}');
+}
+
+export function isCustomMethod(path) {
+  return path.includes(':');
+}
+
+/**
+ * Checks if a resource is a singleton resource based on the paths for the
```

Comment:
Nit: Could we add a comment explaining what 'singleton resource' and 'standard resource' mean for clarity?